app/src/Doctrine/Website/Docs/CodeBlockConsoleRenderer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Docs;
+
+use function count;
+use function ltrim;
+use function rtrim;
+use function sprintf;
+use function substr;
+
+class CodeBlockConsoleRenderer
+{
+    /**
+     * @param string[] $lines
+     */
+    public function render(array $lines) : string
+    {
+        $code = '<div class="console">';
+
+        $consoleLines  = '';
+        $hasDollarSign = false;
+
+        foreach ($lines as $line) {
+            $hasDollarSign = isset($line[0]) && $line[0] === '$';
+
+            $consoleLines .= $this->renderConsoleLine($line, $hasDollarSign);
+        }
+
+        $consoleLines = rtrim($consoleLines, "\n");
+
+        // if console example has only 1 line and no $ explicitely existed put a $ there manually.
+        if (count($lines) === 1 && ! $hasDollarSign) {
+            $consoleLines = sprintf("<span class=\"noselect\">$ </span>%s\n", ltrim($consoleLines, ' '));
```

Comment:
Does `ltrim()` only need to remove an empty space or can you omit the second argument for the function's default?

Reply:
2nd argument can be removed.